Src/Support/Google.Apis.Auth/OAuth2/ServiceAccountCredential.cs

```diff
@@ -379,6 +407,25 @@ private string CreateJwtAccessToken(string authUri, DateTime issueUtc, DateTime
             return CreateAssertionFromPayload(payload);
         }
 
+
+        /// <param name="scopes">A list of the permissions the application requests.</param>
+        /// <param name="issueUtc">The issue time of the JWT.</param>
+        /// <param name="expiryUtc">The expiry time of the JWT.</param>
+        /// <inheritdoc cref="CreateJwtAccessToken(string, DateTime, DateTime)"/>
+        private string CreateJwtAccessToken(IEnumerable<String> scopes, DateTime issueUtc, DateTime expiryUtc)
```

Comment:
I think this would be clearer as CreateJwtAccessTokenForScopes rather than using overloading.

Reply:
updated